server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/Cli.java

```diff
@@ -110,9 +111,11 @@ void initProcessorGroupsReader(Path workspace) {
         processorGroupsResolver = new ProcessorGroupsResolver(new String(Files.readAllBytes(programConfig)), new String(Files.readAllBytes(groupsConfig)));
       } catch (IOException e) {
         LOG.error("Processor group configuration read error", e);
+        throw new IOException(e.getMessage());
```

Comment:
Looks like we are missing the stack, does `throw e` also work here?

Reply:
Implemented in latest commit.